### PR TITLE
Add student info page back-end

### PR DESCRIPTION
**Pull request by vladisa88**

### CODE REVIEW

File: back-end/src/lms/filters/punishments.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+    DateFilter,
+    NumberFilter,
+    BooleanFilter,
+)
 
 from lms.models.punishments import Punishment
 
 
 class PunishmentFilter(FilterSet):
+    removed = BooleanFilter(method='filter_by_removed_punishment',)
```
Comment (gakhromov) [inline back-end/src/lms/filters/punishments.py:13]: Remove comma

File: back-end/src/lms/views/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            student=pk, date__range=date_range).values_list("date")
+        absences_dates = [date[0] for date in absences_dates]
+        student_subject_marks = {}
+        subject_dates = {}
+        response = []
+        # find all marks for subject
+        for mark in marks:
+            student_subject_marks.setdefault(mark.lesson.subject.title,
+                                             []).append(*mark.mark)
+        for lesson in lessons:
```
Comment (gakhromov) [inline back-end/src/lms/views/dashboard.py:60]: `*` ?

**Review by gakhromov — CHANGES_REQUESTED** [review]
Address my comments.

**Review by gakhromov — APPROVED** [review]
LGTM

**Review by gakhromov — APPROVED** [review]

**Comment by TmLev** [comment]
Merged in #333.

**Pull request closed without merging**